README.md

```diff
@@ -57,6 +78,46 @@ The upgrade lock is needed to:
 - Temporary switch off "house-keeping" job, which task is removing "unhealthy" members and updating DNS records.
 - Make sure that we are upgrading one cluster member at a time.
 
+Migration of an existing cluster to multiregion setup
+=====================================================
+Step 1: you have to migrate to the multiregion setup but with only 1 (ONE) active region. To do that you need to run:
+
+    senza --region=eu-central-1 update etcd-cluster-multiregion.yaml existingcluster \
+            HostedZone=elephant.example.org \
+            DockerImage=registry.opensource.zalan.do/acid/etcd-cluster:3.0.15-p14 \
+            ActiveRegions=eu-central-1 \
+            InstanceCount=5
+
+And do instance rotation like during Upgrade procedure.
+
+Step 2: Enable the second region.
+
+    senza --region=eu-central-1 update etcd-cluster-multiregion.yaml existingcluster \
+            HostedZone=elephant.example.org \
+            DockerImage=registry.opensource.zalan.do/acid/etcd-cluster:3.0.15-p14 \
+            ActiveRegions=eu-central-1,eu-west-1 \
+            InstanceCount=5
+
+And rotate all instances once again. Although the second region is not there yet, cluster will think that it is working in multiregion mode.
```

Comment:
It is not. As soon a you enabled two regions every instance will start  publishing public dns names in peerURLs and clientURLs. Old instances (not upgraded) will not recognize it, think that it is an enemy and KILL it.